spice/tests/rv_input.py

```diff
@@ -174,10 +173,11 @@ def run(vt_test, test_params, env):
         for i in keys2:
             test.vm_c.send_key(i)
         logged_keys = act.klogger_stop(test.vmi_g, ssn)
-        ssn.close
-        keysyms = map(lambda (ignore, keysym): keysym, logged_keys)
+        ssn.close()
+        keysyms = [key[1] for key in logged_keys]
```

Comment:
Personal opinion: previous variant was more understandable.  
It was clear that we deal with: list of pressed: (keycode, keysym)
But, not now.